docs/04-api-reference/front-commerce-core/config.mdx

```diff
@@ -94,7 +94,39 @@ export default defineConfig({
 
 Object that define app configuration
 
-#### `providers`
+### `serverEvents`
```

Comment:
**thought:** This section is just displaying information without any information of what it does or why it is done.

**suggestion:** Add explanations

**suggestion(alt):** Remove the code examples

Reply:
Fixed with your suggestion